packages/cypress/src/pageObjects/AppMessage/index.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
```

Comment:
since AppMessage is a component inside the RainbowFirebaseApp, we should make a page object for RainbowFirebaseApp